devel/ansible/roles/dev/tasks/main.yml

```diff
@@ -25,63 +25,71 @@
       - pungi
       - python
       - python-alembic
-      - python-arrow
-      - python-bleach
       - python-bugzilla
-      - python-click
-      - python-cornice
-      - python-cornice-sphinx
-      - python-devel
-      - python3-diff-cover
-      - python-dogpile-cache
-      - python-fedora
-      - python-ipdb
-      - python-kitchen
-      - python-librepo
-      - python-mock
       - python-openid
-      - python-pillow
       - python-progressbar
-      - python-psycopg2
       - python-pydns
       - python-pylibravatar
-      - python-pyramid
-      - python-pyramid-mako
       - python-pyramid-fas-openid
-      - python-pyramid-tm
-      - python-rpdb
       - python-simplemediawiki
-      - python-sqlalchemy
       - python-webtest
-      - python-zmq
+      - python2-arrow
+      - python2-bleach
+      - python2-click
       - python2-colander
       - python2-createrepo_c
       - python2-cryptography
+      - python2-devel
       - python2-dnf
+      - python2-dogpile-cache
       - python2-fedmsg-commands
       - python2-fedmsg-consumers
+      - python2-fedora
       - python2-flake8
+      - python2-ipdb
       - python2-jinja2
+      - python2-kitchen
+      - python2-librepo
       - python2-markdown
+      - python2-mock
+      - python2-pillow
+      - python2-psycopg2
+      - python2-pyramid
+      - python2-pyramid-mako
+      - python2-pyramid-tm
       - python2-pytest-cov
+      - python2-rpdb
       - python2-sphinx
+      - python2-sqlalchemy
       - python2-sqlalchemy_schemadisplay
       - python2-waitress
       - python2-yaml
+      - python2-zmq
```

Comment:
Out of curiosity, why do you need to directly depend on zmq? fedmsg should depend on it.

Reply:
I'm not sure why ```zmq``` is here, but I just tested the Vagrant box without it and it worked fine, so I'll drop it.